spec/models/partner_spec.rb

```diff
@@ -114,6 +114,20 @@
     end
   end
 
+  describe "#partials_to_show" do
+    let(:partner) { create(:partner, diaper_bank_id: 100) }
+    it 'has 9 partials when there are no displayable partials configured' do
+      expect(partner.partials_to_show.size).to eq(9)
```

Comment:
```suggestion
      expect(partner.partials_to_show).to eq(Partner::ALL_PARTIALS)
```

^ this might make the test a bit more robust, so when you add a new field you won't have to worry about fixing this spec.